src/main/java/time/MonthView.java

```diff
@@ -93,10 +93,23 @@ private void printTasksForWeek(LocalDate weekStart, int maxTasks, TaskManager ta
         }
     }
 
+
     private void printTaskForDay(List<Task> dayTasks, int taskIndex) {
         if (taskIndex < dayTasks.size()) {
             Task task = dayTasks.get(taskIndex);
-            System.out.printf(uiRenderer.TASK_DISPLAY_FORMAT, task.getName());
+            switch (task.getTaskType()) {
+            case "E":
+                System.out.printf(uiRenderer.TASK_DISPLAY_FORMAT, "*");
+                break;
+            case "D":
+                System.out.printf(uiRenderer.TASK_DISPLAY_FORMAT, "#");
+                break;
+            case "T":
+                System.out.printf(uiRenderer.TASK_DISPLAY_FORMAT, "+");
+                break;
+            default:
+                break;
+            }
```

Comment:
Good idea to use placeholder symbols! Would it be clearer to the user if E, D and T represent event, deadline and todo respectively rather than the symbols?

Reply:
Seconded on this. *, # and + are kind of confusing even if we do specify them in the UG.

Reply:
I was working to make the user get a better view of what days are free by just a glance at his monthview.

But I do agree that we need a lookup table for the person to understand.

I tried to color code each type of task, but that hinders the formatting, so I settled on plain text